app/routers/categories.py

```diff
@@ -46,39 +49,48 @@ def get_categories(request: Request,
                                    f"unallowed params.")
 
 
-@router.get("/list")
-def get_all_categories(
-        db_session: Session = Depends(get_db)) -> List[Category]:
-    return db_session.query(Category).all()
-
-
 @router.get("/")
-def get_categories_by_user_id(
-        user_id: int, db_session: Session = Depends(get_db)
-) -> List[Category]:
-    return get_user_categories(db_session, user_id)
+def category_color_insert(request: Request) -> _TemplateResponse:
+    return templates.TemplateResponse("categories.html", {
+        "request": request
+    })
 
 
 # TODO(issue#29): get current user_id from session
 @router.post("/")
-async def set_category(category: CategoryModel,
-                       db_sess: Session = Depends(get_db)) -> Dict[str, Any]:
-    if not validate_color_format(category.color):
+async def set_category(request: Request,
+                       name: str = Form(None),
+                       color: str = Form(None),
+                       db_sess: Session = Depends(get_db)):
+
+    message = ""
+    user_id = 1    # until issue#29 will get current user_id from session
+    color = color.replace('#', '')
+    if not validate_color_format(color):
         raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST,
-                            detail=f"Color {category.color} if not from "
+                            detail=f"Color {color} if not from "
                                    f"expected format.")
     try:
-        cat = Category.create(db_sess,
-                              name=category.name,
-                              color=category.color,
-                              user_id=category.user_id)
+        Category.create(db_sess,
+                        name=name,
+                        color=color,
+                        user_id=user_id)
     except IntegrityError:
         db_sess.rollback()
-        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST,
-                            detail=f"category is already exists for "
-                                   f"user {category.user_id}.")
-    else:
-        return {"category": cat.to_dict()}
+        message = "Category already exists"
+        return templates.TemplateResponse("categories.html", {
+            "request": request,
+            "message": message,
+            "name": name,
+            "color": color,
```

Comment:
Maybe we should create the dictionary in another function? (It takes lot of space)

Reply:
fix :)